jobrunner/config.py

```diff
@@ -51,6 +51,9 @@
 
 TEMP_DATABASE_NAME = os.environ.get("TEMP_DATABASE_NAME")
 
+PRESTO_TLS_KEY = os.environ.get("PRESTO_TLS_KEY")
+PRESTO_TLS_CERT = os.environ.get("PRESTO_TLS_CERT")
```

Comment:
Just a thought: do we want to pass in paths here and have `config.py` read their contents. I suppose it would minimise the amount of time these values spend floating around in environment variables, and would mean that we don't have them on screen every time we edit the `.env` file.

Reply:
That's a good idea

Reply:
It also doesn't leave them in the processes environment table while its running